services/src/main/java/org/keycloak/services/resources/admin/AdminMessageFormatter.java

```diff
@@ -1,65 +0,0 @@
-/*
```

Comment:
Ah, so we already had a way to format messages in a localized manner on the API side. Sigh, I really wish we were using this instead of returning arbitrary keys.